valheim-plus-updater

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Include defaults and common functions
+. /usr/local/etc/valheim/defaults
+. /usr/local/etc/valheim/common
+
+main() {
+    create_directory_structure
+    check_for_update
+}
+
+create_directory_structure() {
+    mkdir -p "$vp_download_path" "$vp_install_path"
+}
+
+check_for_update() {
+    cd "$vp_download_path"
+    local api_response=$(curl -sL https://api.github.com/repos/nxPublic/ValheimPlus/releases/latest | jq -r ".assets[] | select(.name == \"$vp_zipfile\")")
```

Comment:
Just a comment on this, GitHub API is rate limited for unauthenticated requests. Realistically, no one should hit it as the limit is 60 requests an hour but maybe something we should add in the README under the Updates section?

Reply:
Great idea! Will do.